src/main/java/Gops/TaskList.java

```diff
@@ -0,0 +1,174 @@
+package Gops;
+
+import java.util.ArrayList;
+public class TaskList {
+    private ArrayList<Todo> taskList;
+
+    /**
+     * Creates new task list
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<Todo>();
+    }
+
+    /**
+     * Returns the number of tasks in the list
+     * @return number of tasks
+     */
+    public int numberOfTasks() {
+        return taskList.size();
+    }
+
+    /**
+     * Adds new todo to the task list and amends status
+     * @param todoDescription task description
+     * @param status task status
+     */
+    public void addNewTodo(String todoDescription, boolean status) {
```

Comment:
You might achieve better encapsulation if `TaskList` is agnostic to the types of tasks it can contain. So a general method of `addTask` may be more suitable. Though this is entirely up to your discretion.